src/main/java/eu/dissco/backend/repository/AnnotationRepository.java

```diff
@@ -46,14 +46,14 @@ public List<JsonApiData> getAnnotationsForUserJsonResponse(String userId, int pa
         .limit(pageSize).offset(offset).fetch(this::mapToJsonApiData);
   }
 
-  public AnnotationResponse getAnnotation(String id) {
+  public JsonApiData getAnnotation(String id) {
     return context.select(NEW_ANNOTATION.asterisk())
         .from(NEW_ANNOTATION)
         .where(NEW_ANNOTATION.ID.eq(id))
-        .fetchOne(this::mapToAnnotation);
+        .fetchOne(this::mapToJsonApiData);
   }
 
-  public List<AnnotationResponse> getAnnotations(int pageNumber, int pageSize) {
+  public List<AnnotationResponse> getAnnotationsObject(int pageNumber, int pageSize) {
```

Comment:
Is it correct that some still use the 'old' objects and some the new?

Reply:
Not this function, but the function `getForTargetObject()` is used by the the specimenService and digitalMediaObjectService*. The "old" object is useful because we return a json with values for a specimen/mediaObject plus with its annotations. See [`getDigitalMediaObjectFull()`](https://github.com/DiSSCo/dissco-core-backend/blob/be8faab704fd6f6ffe616d699b56d59bd3cab09b/src/main/java/eu/dissco/backend/service/DigitalMediaObjectService.java#L44)  and [`getSpecimenByIdFull()`](https://github.com/DiSSCo/dissco-core-backend/blob/be8faab704fd6f6ffe616d699b56d59bd3cab09b/src/main/java/eu/dissco/backend/service/SpecimenService.java#L97)

(Well actually the function `getForTargetObject()` is used by the annotationService, which is called by the other services, but that's semantics)